Zend/zend_exceptions.c

```diff
@@ -772,21 +773,40 @@ void zend_register_default_exception(void) /* {{{ */
 		}
 	} ZEND_HASH_FOREACH_END();
 
-	INIT_CLASS_ENTRY(ce, "ErrorException", error_exception_functions);
-	error_exception_ce = zend_register_internal_class_ex(&ce, default_exception_ce);
-	error_exception_ce->create_object = zend_error_exception_new;
-	zend_declare_property_long(error_exception_ce, "severity", sizeof("severity")-1, E_ERROR, ZEND_ACC_PROTECTED);
+	INIT_CLASS_ENTRY(ce, "Error", NULL);
```

Comment:
Yes, but take Symfony2 for example.
You can find such code :

use Foo\Bar\String;

It gives : "Fatal error : class String is reserved." when it gets compiled, because the compiler checks for reserved words when use alias are resolved, at compile time.

So I guess some people may have written : use Foo\Bar\Error;  and this will break as well.